open_spiel/python/egt/alpharank.py

```diff
@@ -791,3 +791,41 @@ def compute(payoff_tables,
     print_results(payoff_tables, payoffs_are_hpt_format, rhos, rho_m, c, pi)
 
   return rhos, rho_m, pi, num_profiles, num_strats_per_population
+
+def suggest_alpha(payoff_tables):
+  """Suggests an alpha for use in alpha-rank. 
+
+  The suggested alpha is approximately the smallest possible alpha such that 
+  the ranking has 'settled out'. It is calculated as 2/min_gap_between_payoffs. 
```

Comment:
Would you mind adding a line or two of added context here (similar to the transition probability series explanation you provided in the main thread), in case users are interested in where this derivation comes from? Thanks ahead!